src/simulator/simulator.cpp

```diff
@@ -498,11 +526,13 @@ void Simulator::Run() {
   publishTransform();
 
   if (FLAGS_localize) {
```

Comment:
Something odd about some of the publishers being functions and others being embedded in the Run/Update functions.

Reply:
Agreed. Though it was written this way previously I have now refactored this into a separate function to mirror the rest of the code.